core-lightning/umbrel-app.yml

```diff
@@ -8,25 +8,20 @@ description: >-
   Get started with the Lightning network today with Core Lightning - a
   Lightning Network implementation focusing on spec compliance and performance.
 
+
   The Lightning Network allows ultra cheap and almost instant Bitcoin transactions.
   By running a Lightning node, you can not only self-custody your Bitcoin on
   Lightning, but also earn sats by routing payments on the network.
 
 
-  Connect Zeus or any other wallet that supports Core Lightning's REST interface
-  to remotely manage and access your node from anywhere.
-
-
-  Powered by Core Lightning from Elements.
+  Core Lightning and Application powered by Blockstream.
 
-
-  An official app from Umbrel.
-developer: Umbrel
-website: https://umbrel.com
+developer: Shahana Farooqui
```

Comment:
Are you okay to change this to Core Lightning to match the `submitter` field so that it is super clear to users that this is the official Core Lightning app?